bazel/cert.bzl

```diff
@@ -0,0 +1,130 @@
+"""
+This module contains functions to generate a simple CA
+"""
+
+# buildifier: disable=function-docstring-args
+def _redpanda_private_key(name, certificate):
+    private_key = certificate + ".key"
+
+    pr_key_gen = name + "_key_gen"
+    native.genrule(
```

Comment:
These macros are awesome - thank you for adding them. No need to change anything but generally it's recommended to use run_binary over genrule because run_binary is much more strict about inputs and thus can be cached better.

https://github.com/bazelbuild/bazel-skylib/blob/main/docs/run_binary_doc.md